media/css/firefox/os/firefox-os-new.less

```diff
@@ -0,0 +1,491 @@
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+@import "../../sandstone/lib.less";
+
+/*-------------------------------------------------------------------------*/
+// Common
+
+#outer-wrapper {
+    .aurora-bars();
+}
+
+#wrapper {
+    width: 100%;
+    padding-bottom: 0;
+
+    * {
+        .border-box();
+    }
+}
+
+section {
+    clear: both;
+    text-align: center;
+
+    h1 {
+        text-shadow: none;
+    }
+
+    h2 {
+        .open-sans-light();
+        .font-size(3rem);
+        line-height: 1.2;
+    }
+
+    p {
+        .font-size(28px);    
+        .open-sans-light();
+        letter-spacing: -1px;
+        line-height: 1.2;
+    }
+
+    .content-wrapper {
+        margin: 0 auto;
+        width: 100%;
+        max-width: 1100px;
+        padding: 80px 0; 
+    }
+}
+
+/*-------------------------------------------------------------------------*/
+// Sections
+
+.fxos-features,
+.fxos-try {
+    background-color: #fff;
+
+    p, h2, h3 {
+        color: #56565a;
+    }
+}
+.fxos-hero-space,
+.fxos-mdn,
+.fxos-hub {
+    a, h1, h2 {
+        color: #fff;
+        text-shadow: none; 
+    }
+
+    a {
+       .font-size(20px); 
+    }
+}
+
+.fxos-hero-space {
+    position: relative;
+    overflow: hidden;
+
+    .content-wrapper {
+        padding-bottom: 0;
+    }
+
+    .hero-content {
+       margin: auto;
+       margin-bottom: 60px;
+    }
+    h1 {
+        .font-size(84px);
+        margin: 0 auto @baseLine;
+        padding: 0 115px;
+    } 
+    p {
+        color: #bee1f5;
+        .font-size(36px);
+        padding: 0 45px;
+        margin-bottom: 30px;
+    }
+    .hero-image {
+        position: relative;
+        top: 108px;
+        margin-top: -108px;
+    }
+}
+
+.fxos-features {
+    float: left;
+    padding: 0 0 60px 0; 
+    width: 100%;
+
+    p {
+        margin-bottom: 60px;
+        padding: 0 70px;
+    }
+
+    h2 {
+        margin-bottom: 40px;
+    }
+   
+    .features {
+        list-style-type: none;
+        margin: 0 auto;
+        padding: 0;
+        position: relative;
+
+        h3 {
+            .font-size(24px);
+            margin-top: @baseLine;
+        }
+
+        p {
+            color: #999;
+            .font-size(20px);
+            line-height: 1.3;
+            max-width: 420px;
+            margin: @baseLine auto;
+            padding: 0;
+        }
+
+        li {
+            float: left;
+            margin: 0;
+            padding: 140px @gridGutterWidth 0;
+            position: relative;
+            width: 50%;
+
+            &:before {
+                position: absolute;
+                top: 0;
+                left: 50%;
+                margin-left: -70px;
+                width: 140px;
+                height: 140px;
+                content: '';
+                background-image: url(/media/img/firefox/os-new/icons/features-sprite.png);
```

Comment:
No need to include this if you're doing `.at2x` below. The mixin should generate both for you.